hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/scope/HederaNativeOperations.java

```diff
@@ -89,22 +89,6 @@ public interface HederaNativeOperations {
      */
     void finalizeHollowAccountAsContract(@NonNull Bytes evmAddress);
 
-    /**
```

Comment:
I presume this is all done by token service?

Reply:
True.